src/components/ProfileModal.tsx

```diff
@@ -82,6 +85,13 @@ const ProfileModal: React.FC<ProfileModalProps> = ({ isOpen, onClose }) => {
     deployedContracts?: boolean;
   }>({});
 
+  const { value: debouncedGithubHandle, isDebouncing: githubDebouncing } =
+    useDebounce(settings?.socials?.[SocialType.GITHUB], 500);
```

Comment:
Can probably be longer than 500ms tbh. Not changing often. But then you might want to throttle instead.